c2corg_ui/static/js/documentservice.js

```diff
@@ -149,9 +152,7 @@ app.Document.prototype.pushToAssociations = function(doc, doctype,
  * @export
  */
 app.Document.prototype.removeAssociation = function(id, type, event) {
-  var associations = type === 'outings' ?
-          this.document.associations.recent_outings.documents :
-          this.document.associations[type];
+  var associations = this.document.associations[type];
```

Comment:
I have dropped the "outings" exception because, as far as I know, it's not possible to remove an associated outing from any document (except now from images with this PR). It's only possible to remove an associated route from the outing detail view page.